src/main/java/org/mdbenefits/app/submission/actions/ReformatHouseholdNameBeforeSaving.java

```diff
@@ -0,0 +1,30 @@
+package org.mdbenefits.app.submission.actions;
+
+import formflow.library.config.submission.Action;
+import formflow.library.data.FormSubmission;
+import formflow.library.data.Submission;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ReformatHouseholdNameBeforeSaving implements Action {
+    @Override
+    public void run(Submission submission, String id) {
+        ArrayList<HashMap<String, Object>> householdSubflow = (ArrayList<HashMap<String, Object>>) submission.getInputData().get("household");
+        if (householdSubflow != null) {
+            HashMap<String, Object> householdMember = householdSubflow.stream()
+                .filter(iteration -> id.equals(iteration.get("uuid"))).toList().get(0);
+            String firstName = (String) householdMember.remove("householdMemberFirstName");
```

Comment:
Currently, empty fields are have an empty string but in the future we should use getOrDefault and only save if the string is not empty.